### PR TITLE
use UCrop features

### PR DESCRIPTION
**Pull request by omegasoft7**

### CODE REVIEW

File: rx_paparazzo/src/main/java/com/fuck_boilerplate/rx_paparazzo/RxPaparazzo.java [hunk 1]
```diff
@@ -107,6 +108,14 @@ public BuilderImage(T ui) {
         }
 
         /**
+         * Call it when crop features is required.
+         */
+        public BuilderImage<T> cropFeatures(UCropFeature feature) {
```
Comment (VictorAlbertos) [inline rx_paparazzo/src/main/java/com/fuck_boilerplate/rx_paparazzo/RxPaparazzo.java]: This method should be named `crop`, being that way an overloaded version of existing `crop`.

File: rx_paparazzo/src/main/java/com/fuck_boilerplate/rx_paparazzo/RxPaparazzo.java [hunk 2]
```diff
@@ -157,6 +166,14 @@ public BuilderImages(T ui) {
         }
 
         /**
+         * Call it when crop features is required.
+         */
+        public BuilderImages<T> cropFeatures(UCropFeature feature) {
```
Comment (VictorAlbertos) [inline rx_paparazzo/src/main/java/com/fuck_boilerplate/rx_paparazzo/RxPaparazzo.java]: This method should be named `crop`, being that way an overloaded version of existing `crop`.

**Comment by VictorAlbertos** [comment]
Sorry, I can't accept this PR. There are several points, but the main one is that we are currently exposing UCrop api (UCrop.Options) to configure the crop. With this PR we would have another class, which has a wrapper around some features of UCrop, but it does not expose the UCrop api itself. 

And I would prefer the configuration using just one method, without having to chain another one when building the instance. Indeed, I'm not sure if this will work.

In short, I think that with this PR the RxPaparazzo Api has lost some quality. 

Let me see if I can implement the requested feature using the approach intent which I've told you before ;)
